digitalpublicgoods/synthea.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Synthea",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/synthetichealth/synthea#license"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/synthetichealth/synthea/wiki"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
```

Comment:
This is true if all data is generated locally offline. I don't see something indicating data is shared back to a central server or database. Is that assumption incorrect?

Reply:
Data is not shared back to a server or database, and the software doesn't collect any metrics or telemetry information to share back with the project. Once installed, the software is completely standalone and does not require an internet connection. We have no idea how people are using it, except when they choose to tell us.

Happy to answer other questions. Disclosure: I am one of the project maintainers.

Reply:
@jawalonoski Thanks for the clarification!